docs/references/strategies/languages/python/pyproject-generic.md

```diff
@@ -0,0 +1,62 @@
+# PyProject.toml Generic Support
+
+The FOSSA CLI provides robust support for analyzing Python projects that use the [PEP 621](https://peps.python.org/pep-0621/) `pyproject.toml` format, which is becoming the standard for Python project metadata and configuration.
+
+## Project Discovery
+
+The CLI searches for `pyproject.toml` files and automatically detects which build system is being used by examining the file contents.
+
+## Project Type Detection
+
+When analyzing a `pyproject.toml` file, the CLI will attempt to determine which build system is in use by checking for specific sections:
+
+1. **Poetry**: Detected by the presence of `[tool.poetry]` section
+2. **PDM**: Detected by the presence of `[tool.pdm]` section
+3. **PEP 621**: Detected by the presence of `[project]` section (standard PEP 621 project)
+4. **Unknown**: If none of the above sections are found
```

Comment:
What is the difference for the customer between PEP-621 and Unknown? 

In the "Analysis Strategies" section the "Unknown" project type disappears.